src/components/VersionSelector.js

```diff
@@ -0,0 +1,161 @@
+      break;
+    case 'colosseum':
+      text = ['Colosseum'];
+      color = [colors.gameVersion.colosseum];
+      break;
+    case 'xd':
+      text = ['XD'];
+      color = [colors.gameVersion.xd];
+      break;
+    case 'black-2-white-2':
+      text = ['Black', 'White'];
+      color = [colors.gameVersion.black, colors.gameVersion.white];
+      break;
+    case 'x-y':
+      text = ['X', 'Y'];
+      color = [colors.gameVersion.X, colors.gameVersion.Y];
+      break;
+    case 'omega-ruby-alpha-sapphire':
+      text = ['Omega Ruby', 'Alpha-Sapphire'];
+      color = [colors.gameVersion.omegaRuby, colors.gameVersion.alphaSapphire];
+      break;
+    case 'sun-moon':
+      text = ['Sun', 'Moon'];
+      color = [colors.gameVersion.sun, colors.gameVersion.moon];
+      break;
+    case 'ultra-sun-ultra-moon':
+      text = ['Ultra Sun', 'Ultra Moon'];
+      color = [colors.gameVersion.ultraSun, colors.gameVersion.ultraMoon];
+      break;
+    case 'lets-go':
+      text = ["Let's Go Pikachu", "Let's Go Eevee"];
+      color = [colors.gameVersion.letsGoPikachu, colors.gameVersion.letsGoEevee];
+      break;
+    case 'sword-shield':
+      text = ['Sword', 'Shield'];
+      color = [colors.gameVersion.sword, colors.gameVersion.shield];
+      break;
+    default:
+      text = ['NULL', 'NULL'];
+      color = ['red', 'blue'];
+      break;
+  }
+
+  return (
+    <View styles={styles.container}>
+      {text.length > 1 && (
+        <View style={styles.textContainer}>
+          <View style={[styles.textContainer1, { backgroundColor: color[0] }]}>
+            <Text style={styles.text} textShadowColor>
+              {text[0]}
+            </Text>
+          </View>
+          <View style={[styles.textContainer2, { backgroundColor: color[1] }]}>
+            <Text style={styles.text}>{text[1]}</Text>
+          </View>
+        </View>
+      )}
+      {text.length == 1 && (
+        <View style={styles.textContainer}>
+          <View
+            style={[
+              styles.textContainer1,
+              {
+                borderTopRightRadius: 100,
+                borderBottomRightRadius: 100,
+                backgroundColor: color[0],
+              },
+            ]}
+          >
+            <Text style={styles.text}>{text[0]}</Text>
+          </View>
+        </View>
+      )}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    height: 'auto',
+    padding: 8,
+    borderRadius: 20,
+    elevation: 8,
+    flexDirection: 'column',
+  },
+  textContainer: {
+    flexDirection: 'row',
+  },
+  textContainer1: {
+    flex: 1,
+    borderTopLeftRadius: 100,
+    borderBottomLeftRadius: 100,
+    backgroundColor: 'red',
+  },
+  textContainer2: {
+    flex: 1,
+    borderTopRightRadius: 100,
+    borderBottomRightRadius: 100,
+  },
+  text: {
+    textAlign: 'center',
+    paddingVertical: 5,
+    color: 'white',
+    textShadowColor: 'black',
+    textShadowRadius: 10,
```

Comment:
Instead of this shadow (here and in other lines of code), try to add the black stroke outside the captions - just like we have it in Figma.

Reply:
This can be done only by cloning same text multiple times with different shadow direction. There is no prop like stroke in text component.
https://reactnative.dev/docs/text-style-props